doc/intro.rst

```diff
@@ -31,10 +31,11 @@ hardware platform, hardware configuration, OS, build settings (e.g.
 regular compilers vcs optimized compilers), etc. Results are always
 organized by environments, so we can compare apples to apples.
 
-Projects can have subscriptions, which are email address that should be
-notified about important events such as changing test results. ProjectStatus
-records the most recent build of a project, against which future results should
-be compared in search for important events to notify subscribers about.
+Projects can have subscriptions, which are users or manual email addresses that
```

Comment:
I suggest to phrase this as "which are either users of manually-entered email addreses that ..."